<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Relay</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <!-- LEFT PANEL: Available Devices -->
     <widget class="QWidget" name="leftPanel">
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QLabel" name="labelAvailable">
         <property name="text">
          <string>Available Devices</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollAvailable">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAvailableContents">
          <layout class="QVBoxLayout" name="availableList"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <!-- RIGHT PANEL: Source, Targets, Controls -->
     <widget class="QWidget" name="rightPanel">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QLabel" name="labelSource">
         <property name="text">
          <string>Source Device</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="sourceContainer">
         <layout class="QVBoxLayout" name="sourceLayout"/>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTargets">
         <property name="text">
          <string>Target Devices</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollTarget">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollTargetContents">
          <layout class="QVBoxLayout" name="targetList"/>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="startStopButton">
           <property name="text">
            <string>Start Relay</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearTargetsButton">
           <property name="text">
            <string>Clear Targets</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="relayStatusLabel">
         <property name="text">
          <string>Relay Status: Stopped</string>
         </property>
         <property name="styleSheet">
          <string notr="true">color: red;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
